api/api.go

```diff
@@ -317,6 +317,8 @@ type CloudBackupRestoreRequest struct {
 }
 
 type CloudBackupGroupCreateResponse struct {
+	// ID for this group of backups
+	GroupCloudBackupID string
 	// Names of the tasks performing this group backup
 	Names []string
```

Comment:
Change this to IDs too?

Reply:
AFAIK, NO. This will need capability on clients side to interpret both old and new response structures( just like we do on server side for requests)